src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Random random = new Random();
+        int randomNumber = random.nextInt(Colors.values().length);
+        return Colors.values()[randomNumber].toString();
```

Comment:
Instead of using `toString()` to get the string representation of the enum constant, use the `name()` method. This is recommended to avoid potential issues if `toString()` is overridden in the future. Refer to the checklist: 'Use name() for getting String representation of enum constants'.